### ISSUE TITLE
Layout/MultilineMethodCallIndentation: 'method_on_receiver_last_line?': undefined method 'line' for nil (NoMethodError)

### ISSUE THREAD

**Issue by r-mllr** [rubocop/rubocop#14843]
## Steps to reproduce the problem

When I run `bundle exec rubocop -d rubocop_test.rb`, I get an `undefined method 'line' for nil (NoMethodError)`. When I run the same command, but with `--force-default-config`, there's no error message. Hence, I believe it has to do with 

```
EnforcedStyle: 'indented_relative_to_receiver'
```

### Minimal example file `rubocop_test.rb`

```ruby
# frozen_string_literal: true

# Just a test class
class TestController < ApplicationController
  def show
    params[:author_type]
      .constantize
  end
end
```

### `.rubocop.yml`

```
AllCops:
  NewCops: enable
  TargetRubyVersion: 3.3

Layout/MultilineMethodCallIndentation:
  EnforcedStyle: 'indented_relative_to_receiver'
```

## Expected behavior

There is no error message when linting above example file.

## Actual behavior

An error is raised

```
An error occurred while Layout/MultilineMethodCallIndentation cop was inspecting /home/r-mllr/tmp_rubocop_test/rubocop_test.rb:6:4.
For /home/r-mllr/tmp_rubocop_test: configuration from /home/r-mllr/tmp_rubocop_test/.rubocop.yml
Default configuration from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/config/default.yml
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /home/r-mllr/tmp_rubocop_test/rubocop_test.rb
/home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/layout/multiline_method_call_indentation.rb:326:in `method_on_receiver_last_line?': undefined method `line' for nil (NoMethodError)

            node.loc.dot.line == base_receiver.last_line &&
                        ^^^^^
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/layout/multiline_method_call_indentation.rb:302:in `alignment_base_for_chained_receiver?'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/layout/multiline_method_call_indentation.rb:292:in `find_hash_method_base_in_receiver_chain'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/layout/multiline_method_call_indentation.rb:281:in `receiver_alignment_base'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/layout/multiline_method_call_indentation.rb:246:in `alignment_base'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/layout/multiline_method_call_indentation.rb:169:in `check_regular_indentation'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/layout/multiline_method_call_indentation.rb:134:in `offending_range'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/mixin/multiline_expression_indentation.rb:22:in `on_send'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:107:in `public_send'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:107:in `block (2 levels) in trigger_responding_cops'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:171:in `with_cop_error_handling'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:106:in `block in trigger_responding_cops'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:105:in `each'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:105:in `trigger_responding_cops'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:69:in `on_send'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.49.0/lib/rubocop/ast/traversal.rb:164:in `on_def'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:71:in `on_def'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.49.0/lib/rubocop/ast/traversal.rb:164:in `on_class'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:71:in `on_class'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.49.0/lib/rubocop/ast/traversal.rb:20:in `walk'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/commissioner.rb:87:in `investigate'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/team.rb:174:in `investigate_partial'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cop/team.rb:108:in `investigate'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:354:in `block in inspect_file'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:353:in `each'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:353:in `flat_map'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:353:in `inspect_file'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:295:in `block in do_inspection_loop'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:330:in `block in iterate_until_no_changes'
	from <internal:kernel>:187:in `loop'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:323:in `iterate_until_no_changes'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:291:in `do_inspection_loop'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:171:in `block in file_offenses'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:196:in `file_offense_cache'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:170:in `file_offenses'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:158:in `process_file'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:139:in `block in each_inspected_file'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:138:in `each'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:138:in `reduce'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:138:in `each_inspected_file'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:124:in `inspect_files'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/runner.rb:77:in `run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli/command.rb:11:in `run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli/environment.rb:18:in `run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli.rb:128:in `run_command'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli.rb:135:in `execute_runners'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli.rb:54:in `block in run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli.rb:87:in `profile_if_needed'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/lib/rubocop/cli.rb:45:in `run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/rubocop-1.84.1/exe/rubocop:15:in `<top (required)>'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/rubygems.rb:303:in `load'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/rubygems.rb:303:in `activate_and_load_bin_path'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/bin/rubocop:25:in `<top (required)>'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/cli/exec.rb:61:in `load'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/cli/exec.rb:61:in `kernel_load'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/cli/exec.rb:24:in `run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/cli.rb:500:in `exec'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor.rb:538:in `dispatch'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/cli.rb:35:in `dispatch'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/cli.rb:29:in `start'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/bundler-4.0.3/exe/bundle:28:in `block in <top (required)>'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/bundler/friendly_errors.rb:118:in `with_friendly_errors'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/gems/3.3.0/gems/bundler-4.0.3/exe/bundle:20:in `<top (required)>'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/rubygems.rb:303:in `load'
	from /home/r-mllr/.rbenv/versions/3.3.10/lib/ruby/site_ruby/3.3.0/rubygems.rb:303:in `activate_and_load_bin_path'
	from /home/r-mllr/.rbenv/versions/3.3.10/bin/bundle:25:in `<main>'

1 error occurred:
An error occurred while Layout/MultilineMethodCallIndentation cop was inspecting /home/r-mllr/tmp_rubocop_test/rubocop_test.rb:6:4.
Errors are usually caused by RuboCop bugs.
Please, update to the latest RuboCop version if not already in use, and report a bug if the issue still occurs on this version.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.84.1 (using Parser 3.3.10.1, rubocop-ast 1.49.0, analyzing as Ruby 2.7, running on ruby 3.3.10) [x86_64-linux]
.

1 file inspected, no offenses detected
configuration from /home/r-mllr/tmp_rubocop_test/.rubocop.yml
Finished in 0.15538387700507883 seconds
```

## RuboCop version

```
$ bundle exec rubocop -V
1.84.1 (using Parser 3.3.10.1, Prism 1.9.0, rubocop-ast 1.49.0, analyzing as Ruby 3.3, running on ruby 3.3.10) [x86_64-linux]
```
